package/MDAnalysis/visualization/streamlines_3D.py

```diff
@@ -93,7 +99,10 @@ def determine_container_limits(topology_file_path, trajectory_file_path, buffer_
     return tuple_of_limits
 
 
-def produce_grid(tuple_of_limits, grid_spacing):
+# typing: numpy
+def produce_grid(
+        tuple_of_limits: Tuple[float, ...], grid_spacing: float
```

Comment:
```
package/MDAnalysis/visualization/streamlines_3D.py:125: error: Slice index must be an integer or None
```

```suggestion
        tuple_of_limits: Tuple[int, int, int, int, int, int], grid_spacing: float
```

The values in the tuple are used as indices in arrays. They *have to be* integers. 

Also, there need to be 6 items in the tuple otherwise the unpacking later fails. Note that, in theory, `tuple_of_limits` could be any Iterable with 6 items.